ha/k8s_setup/ha_setup.py

```diff
@@ -252,10 +252,10 @@ def process(self):
             # So, to get the cluster_id field from Confstore, we need machine_id
             self._cluster_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}cluster_id')
             # site_id = Conf.get(self._index, f'node{_DELIM}{machine_id}{_DELIM}site_id')
-            self._site_id = '1'
+            self._site_id = NOT_DEFINED
```

Comment:
Can these be defined as constants . They need to be avilable to other components as well for initializing events.
ex: in fault_tolerance/const.py ?

Reply:
@ArchanaLimaye Already it is defined in fault_tolerance/const.py file.